### PR TITLE
Export play-grpc to the play org

### LINKED ISSUE #300: Add Akka gRPC Play docs in the Play documentation
Split out from #209 and #208 as I think this is hard before having an up to date Akka gRPC artifact published.

I think we could easily just port the docs that are now in the Akka gRPC docs to the Play framework docs system.

### LINKED ISSUE #362: Move Play gRPC tests into Play repos
Once we're happy that the test APIs created in #356 and #361 are stable we'll want to move these tests into Play repositories. Some code will be moved into playframework/playframework, some into playframework/scalatestplus-play and some could move into a new repository like playframework/play-grpc. For now we can leave the code in the akka-grpc repository, since it's easier to develop it within a single repository.

### PR DESCRIPTION
**Pull request by dwijnand**
This code has been exported out of the akka-grpc repository into
https://github.com/playframework/play-grpc.

The last release from this repo was:

    com.lightbend.akka.grpc : akka-grpc-play-testkit : 0.4.2

This is now available from the play-grpc repo as:

    com.lightbend.play : play-grpc-testkit : 0.5.0-M5

(Still need to sync it from JCenter to Maven Central.)

... and I just realised that I need to change the package from
akka.grpc.play to something else... Will do!

Tracked as https://github.com/playframework/play-grpc/issues/12

Fixes #362 
Fixes #300

### CODE REVIEW

**Comment by dwijnand** [comment]
Need to fix the docs:

```
Error writing content for page play-framework.html: Unknown snippet [/localhome/jenkinsakka/workspace/akka-grpc-pr-validator/play-interop-test-scala/src/main/protobuf/helloworld.proto (No such file or directory)] referenced from [play-framework.html]
```

**Comment by dwijnand** [comment]
Hmm, @raboof should the play interop projects live here or in play-grpc?

**Comment by raboof** [comment]
@dwijnand I think it makes sense to have them in play-grpc, WDYT?

**Comment by dwijnand** [comment]
@raboof are `PlayScalaClientCodeGenerator` and friends staying here?

My goal is trying to figure out how to land this PR ASAP, and it's broken due to snippets that I removed.

**Comment by raboof** [comment]
Eventually I think it'd be neat if `PlayScalaClientCodeGenerator` moved to `play-grpc` as well, since you'd expect the akka-grpc API's to stabilize and want to evolve the Play codegen along with the other play tools. I would be fine with postponing that work if it makes it easier to make smaller increments, though.

File: docs/src/main/paradox/buildtools/gradle.md [hunk 1]
```diff
@@ -117,4 +117,4 @@ The server can then be started from the command line with:
 
 ## Play Framework support
 
-See the @ref[Play Framework section of the docs](../play-framework.md) for details.
+See the [Play gRPC docs](https://github.com/playframework/play-grpc/blob/docs/docs/src/main/paradox/play-framework.md) for details.
```
[the remark this answers is not in the document]
Comment (dwijnand) [reply docs/src/main/paradox/buildtools/gradle.md]: Thanks, and yes.

**Comment by dwijnand** [comment]
Next steps from here is updating the quickstarts.

**Pull request merged**